### DISCUSSION TITLE
Implementing Tool 2.5 of GPG for Traffic Flows

### DISCUSSION THREAD

**Discussion by ioannides-dB** [Universite-Gustave-Eiffel/NoiseModelling#333] [Ideas]
The OSM import tool uses Tool 2.5 from the Good Practice Guide for Strategic Noise Mapping and the Production of Associated Data on Noise Exposure Version 2 13th January 2006 which produces traffic flow per period (day-evening-night). The CNOSSOS-EU formula though requires a yearly average per hour flow. There is a discrepancy between the GPG and the implementation of Tool 2.5 in NoiseModelling worth looking into as it seems the emissions are currently calculated on the per period flows rather than hourly as specified in CNOSSOS.

PS. See footnote in GPG report for Tool 2.5

**Comment by pierromond** [comment]
I close this discussion because I moved it in Issue tab ;) Thank you !

**Reply by pierromond** [reply]
#334
